packages/angular-output-target/src/output-angular.ts

```diff
@@ -72,16 +72,20 @@ export function generateProxies(
 
   const imports = `/* tslint:disable */
 /* auto-generated angular directive proxies */
-import { ChangeDetectionStrategy, ChangeDetectorRef, Component, ElementRef, NgZone } from '@angular/core';
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, ElementRef, EventEmitter, NgZone } from '@angular/core';
+import { EventEmitter as StencilEventEmitter } from '@stencil/core';
```

Comment:
Ok I see. But there is still the reference to @stencil/core that is problematic, isn’t it?

Reply:
What about using the EventEmitter part of the core generated library? It exposes a `stencil-public-runtime` type definition. So maybe generating something like that:
```
import { EventEmitter as StencilEventEmitter } from 'component-library/dist/types/stencil-public-runtime';
```